aries-site/src/layouts/main/Layout.js

```diff
@@ -12,13 +12,7 @@ const calcPad = size => {
   return val;
 };
 
-export const Layout = ({
-  children,
-  descriptiveHeader,
-  title,
-  isLanding,
-  isNavPage,
-}) => {
+export const Layout = ({ children, descriptiveHeader, title, isLanding }) => {
```

Comment:
I think we can get rid of isLanding as well -- now that everything is full width. Then in `pages/index.js` we can remove the `isLanding` prop from the Layout tag.

Reply:
I wish, seems like isLanding is needed for Footer rendering condition.

Reply:
You're right, good call.